metrics/mape/README.md

```diff
@@ -0,0 +1,133 @@
+---
+title: MAPE
+emoji: 🤗 
+colorFrom: blue
+colorTo: red
+sdk: gradio
+sdk_version: 3.0.2
+app_file: app.py
+pinned: false
+tags:
+- evaluate
+- metric
+description: >-
+  Mean Absolute Percentage Error (MAPE) is the mean percentage error difference between the predicted and actual
+  values.
+---
+
+# Metric Card for MAPE
+
+
+## Metric Description
+
+Mean Absolute Error (MAPE) is the mean of the percentage error of difference between the predicted $x_i$ and actual $y_i$ numeric values:
+
+$$
+M = \frac{1}{n} \sum_{i=1}^{n} \left| \frac{x_i - y_i}{y_i} \right|.
+$$
```

Comment:
Latex is currently not supported in Gradio (see https://github.com/gradio-app/gradio/issues/538). You could either link to a picture of a formula or replace it with a latex free format.